### DISCUSSION TITLE
Is Chatflow load at first conversation?

### DISCUSSION THREAD

**Discussion by kientv** [FlowiseAI/Flowise#1405] [Q&A]
I see that first question is very slow:
- 7s to init retriever
- and more time to finish RAG chain.
Sometimes its not reply at first question after an idle period. 

I'm using:
- Version: 1.4.5
- Chroma or Qdrant
- buffer memory
- Cache LLM
- Cache embedding

How to solve that problem or workaround?

Thanks

**Comment by HenryHengZJ** [comment]
if you use the latest vector store without `Upsert` or `Load Existing`, it should be very fast as it will no longer do the upsert for the 1st question

**Reply by kientv** [reply]
Yep, I'm testing on latest vector store, Do Langchain need to warm-up or re-connect to Vector db?
